fs/io_zenfs.h

```diff
@@ -115,6 +115,7 @@ class ZoneFile {
   void EncodeSnapshotTo(std::string* output) { EncodeTo(output, 0); };
   void EncodeJson(std::ostream& json_stream);
   void MetadataSynced() { nr_synced_extents_ = extents_.size(); };
+  void MetadataUnsynced() { nr_synced_extents_ = 0; };
```

Comment:
This is not being used anywhere and seem to be unrelated to the PR, please remove.

Reply:
Fixed